gigareview/console/tests/acceptance/FetchReports.feature

```diff
@@ -5,6 +5,11 @@ Feature: FetchReports
 
   Scenario: try FetchReports
     Given a EM report is uploaded daily to a sftp server
+    And the database is reset
     When the file is on the sftp server
     And the file ingester has run
-    Then the EM report spreadsheet is downloaded
\ No newline at end of file
+    Then the EM "manuscripts" report spreadsheet is downloaded
+    And the EM "authors" report spreadsheet is downloaded
+    And the EM "reviews" report spreadsheet is downloaded
+    And the EM "reviewers" report spreadsheet is downloaded
+    And the EM "questions" report spreadsheet is downloaded
```

Comment:
These `report spreadsheet is downloaded` statements check that EM reports from date `20220610` have been downloaded from a local SFTP server provided by the `sftp_test` Docker container. Would it be better for all the steps in this acceptance test to just run against the test Docker SFTP server?